src/sentry/api/endpoints/organization_dashboards.py

```diff
@@ -59,15 +59,28 @@ def get(self, request, organization):
                 "-title" if sort_by.startswith("-") else "title",
                 "-date_added",
             ]
+
         elif sort_by in ("dateCreated", "-dateCreated"):
             order_by = "-date_added" if sort_by.startswith("-") else "date_added"
+
+        elif sort_by in ("mostPopular", "-mostPopular"):
+            order_by = [
+                "visits" if sort_by.startswith("-") else "-visits",
+                "-date_added",
+            ]
```

Comment:
nit: we could pull the `sort_by.startswith("-")` out so we dont code repeat, and the conditions are simpler
```suggestion
        if sort_by.startswith("-"):
            sort_by, desc = sort_by[1:], True
        else:
            desc = False
        ....
        elif sort_by == "mostPopular":
            order_by = [
                "-" if desc else "" + "visits",
                "-date_added",
            ]
```

Reply:
nice